pybikes/bysykkelgraphql.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import json
+
+import requests
+
+from . import utils
+from .base import BikeShareStation, BikeShareSystem
+
+
+class BySykkelGraphql(BikeShareSystem):
```

Comment:
~~Is it `Bysykkel` or `BySykkel`?~~ EDIT: My bad, I see that our current code for bysykkel calls it BySykkel. Still, if you do not mind, I would set it as `BysykkelGraphql` and I will change the other one too

Also, the file name should be `bysykkel_graphql.py`